lib/twingly/url.rb

```diff
@@ -56,7 +58,19 @@ def to_addressable_uri(potential_url)
         end
       end
 
-      private :new, :internal_parse, :to_addressable_uri
+      # Workaround for the following bug in addressable:
+      # https://github.com/sporkmonger/addressable/issues/224
+      def addressable_display_uri(addressable_uri)
+        addressable_uri.display_uri
+      rescue ArgumentError => error
+        if error.message.include?("invalid byte sequence in UTF-8")
+          raise Twingly::URL::Error::ParseError
+        end
+
+        raise
+      end
+
+      private :new, :internal_parse, :to_addressable_uri, :addressable_display_uri
```

Comment:
what about splitting this line to multiple lines, so next time we get the nice diff :)